docs/application/dotnet/guides/tizen.circularUI/CircleDateTimeSelector.md

```diff
@@ -0,0 +1,97 @@
+---
+uid: Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector
+summary: CircleDateTimeSelector control guide
+---
+
+# CircleDateTimeSelector
+
+`CircleDateTimeSelector` is a view for Date Picker and Time Picker.
+You can scroll it by [Bezel interaction](https://developer.tizen.org/design/wearable/interaction/bezel-interactions).
+This view is the size that covers the entire screen.
+In order to receive [Rotary event](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/event-handling#rotary), it must be registered as `RotaryFocusObject` property of [CirclePage](xref:Tizen.Wearable.CircularUI.doc.CirclePage).
+
+In the Date type, you can change value of the item by touching the item and rotating bezel to set value of the year: month: day.
+For the Time type, you can change value of the hour: minute: AM / PM.
+
+ |![Date](data/CircleDateTimeSelector_DatePicker.png) | ![Time](data/CircleDateTimeSelector_TimePicker.png)|
+ |:--------------------------------------------------:|:--------------------------------------------------:|
+ |                      Date                          |                           Time                     |
+
+**WARNING: [CircleListView](xref:Tizen.Wearable.CircularUI.doc.CircleListView), [CircleDateTimeSelector](xref:Tizen.Wearable.CircularUI.doc.CircleDateTimeSelector), [CircleScrollView](xref:Tizen.Wearable.CircularUI.doc.CircleScrollView), [CircleStepper](xref:Tizen.Wearable.CircularUI.doc.CircleStepper) must be contained by `CirclePage` or [CircleSurfaceEffectBehavior](xref:Tizen.Wearable.CircularUI.doc.CircleSurfaceEffectBehavior) should be added in [Behaviors](https://developer.xamarin.com/api/type/Xamarin.Forms.Behavior/) of [Page](https://developer.xamarin.com/api/type/Xamarin.Forms.Page/) that contains these controls. If other `pages` contain these controls, it may cause exception or can't display control.**
```

Comment:
Please review the rephrase and check whether the meaning of the content is not changed:

CircleListView, CircleDateTimeSelector, CircleScrollView, CircleStepper must be confined in the CirclePage container or CircleSurfaceEffectBehavior must be added in Behavior class of Page class that contain these controls. If other pages contain these controls, it may cause exception or cannot display the controls.

Reply:
CirclePage isn't a container. So I'll modify it below.
------------------------------------------------------
..., CircleStepper must be confined in the CirclePage component or CircleSurfaceEffectBehavior ...